src/calculateRentalCost.js

```diff
@@ -3,8 +3,30 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_CAR_C0ST = 40;
```

Comment:
There is a typo in the constant name 'RENT_CAR_C0ST'. The '0' should be replaced with an 'O' to correctly spell 'COST'.